### ISSUE TITLE
Project frame hiding

### ISSUE THREAD

**Issue by martskins** [Ruin0x11/intellij-lsp-server#70]
I think the project frame should not be hidden away from the user as a default. Some users might think it's a bug (I did until I found the toggleProjectFrame function).

This seems like an easy fix, if it were to be done. I can definitely submit a PR about this if needed.

**Comment by Ruin0x11** [comment]
Yes, it could be a configuration option instead. Please open a PR if you wish. (However, editing in the IDEA window would currently cause desynchronization between the LSP client and server.)
